docs/excel/custom-functions-troubleshooting.md

```diff
@@ -72,6 +72,10 @@ CustomFunctions.associate("ADD", add);
 
 For more information on this process, see [Associating function names with JSON metadata](../excel/custom-functions-json.md#associating-function-names-with-json-metadata).
 
+## Known issues
+
+See a list of additional known issues by checking the [Excel Custom Functions GitHub repository](https://github.com/OfficeDev/Excel-Custom-Functions/issues).
```

Comment:
I'm not sure these are "additional" issues, rather than a complete list.
```suggestion
Known issues are tracked and reported in the [Excel Custom Functions GitHub repository](https://github.com/OfficeDev/Excel-Custom-Functions/issues).
```